tests/introspection/tests.py

```diff
@@ -170,6 +170,9 @@ def test_get_primary_key_column(self):
     def test_get_indexes(self):
         with connection.cursor() as cursor:
             indexes = connection.introspection.get_indexes(cursor, Article._meta.db_table)
+        if connection.features.can_introspect_index_type:
+            self.assertIn('type', indexes['reporter_id'])
```

Comment:
Can we also add a vendor check for postgresql and assert that the type is 'btree'? Again, this isn't extensible for third-party backends, but we might as well have verification for the built-in backends.